### PR TITLE
Update hrv_OCRFixReplaceList.xml

### PR DESCRIPTION
**Pull request by diomed**

### CODE REVIEW

File: Dictionaries/hrv_OCRFixReplaceList.xml [hunk 1]
```diff
@@ -1427,22 +1448,25 @@
...
     <RegEx find="ebać" replaceWith="ebat ć" />
     <RegEx find="(?&lt;![NnJj])edać" replaceWith="edat ć" />
     <RegEx find="eizbež" replaceWith="eizbjež" />
     <RegEx find="efinišu" replaceWith="efiniraju" />
+    <RegEx find="elokup" replaceWith="jelokup" />
     <RegEx find="erišu" replaceWith="eriraju" />
     <RegEx find="esvest" replaceWith="esvijest" />
     <RegEx find="fanziv" replaceWith="fenziv" />
-    <RegEx find="(mpl|st|f)ikova([nlot])" replaceWith="$1icira$2" />
+    <RegEx find="([mpl|st|f])ikova([nlot])" replaceWith="$1icira$2" />
```
Comment (xylographe) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: `([mpl|st|f])` is equal to `([mpstf|])`, and almost certainly not what you intended. :)

Comment (diomed) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: is there no shortcut for all of these words:
koMPLikovan
sofistikovan, dijagnostifikovan
and all the other wordst that end on Fikovan

Comment (xylographe) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: What was wrong with the original (red) line?
It matches, (spaces added to improve legibility, asterisk presents zero or more letters)
`* MPL ikova [nlot] *`
`* ST ikova [nlot] *`
`* F ikova [nlot] *`

Comment (diomed) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: line was fine. i still sometimes get confused over these things.

**Pull request merged**